src/Atoms/FooterProfileBtn/FooterProfileBtnStyled.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.button`
+  font-size: 26px;
+  color: grey;
```

Comment:
we have fonts and colors as consts in styles folder

Reply:
Fixed.